dbtwiz/commands/build.py

```diff
@@ -0,0 +1,116 @@
+from datetime import date, timedelta
+import json
+import os
+from pathlib import Path
+
+from google.cloud import storage
+
+from dbtwiz.auth import ensure_auth
+from dbtwiz.config import project_config, project_dbtwiz_path
+from dbtwiz.dbt import dbt_invoke
+from dbtwiz.logging import info, debug, error, fatal
+from dbtwiz.manifest import Manifest
+
+
+VALID_TARGETS = ["dev", "build", "prod-ci", "prod"]
+
```

Comment:
There are no line breaks between the global variables in `backfill.py`. Is there a reason behind having it here?

Reply:
It should be added in `backfill.py` and not removed here, since empty lines between things that are not strongly related helps with readability.